src/status_im/ui/screens/desktop/main/chat/views.cljs

```diff
@@ -96,7 +96,7 @@
     [react/text {:style           (styles/message-text message)
                  :selectable      true
                  :selection-color (if outgoing colors/white colors/hawkes-blue)}
-     text]
+     (clojure.string/escape text {\< "&lt;", \> "&gt;", \& "&amp;"})]
```

Comment:
Text styling in `react-native-desktop` implemented via html, so non-escaped html-related characters can break it. Probably we can move your fix to [react/text] component to avoid similar issues in other texts?

Reply:
yep, sounds good

Reply:
on the other hand i would move the fix to `react-native-desktop`, because current behaviour differs from mobile versions